hack/tilt/kubetail.yaml

```diff
@@ -430,6 +430,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
+        - name: OTEL_EXPORTER_OTLP_HEADERS
+          value: "Authorization=<get-api-key-from-hyperdx-ui>"
```

Comment:
Does HyperDX require you to obtain a new API key every time the app is installed? It would be convenient if we could hard code an API key here or disable authentication altogether.